code/lib/components/src/Button/Button.tsx

```diff
@@ -41,13 +41,13 @@ const ButtonWrapper = styled.button<{
 
     svg: {
       display: 'inline-block',
-      height: small ? 14 : 16,
-      width: small ? 14 : 16,
+      height: small ? 12 : 14,
+      width: small ? 12 : 14,
 
       verticalAlign: 'top',
       marginRight: small ? 4 : 6,
-      marginTop: small ? -1 : -2,
-      marginBottom: small ? -1 : -2,
+      marginTop: small ? 0 : -1,
```

Comment:
These changes caused the snapshots to me different FYI @MichaelArestad

Reply:
thanks for the heads up Yann! @MichaelArestad and I changed this intentionally